### PR TITLE
Mobile Work section Project

### PR DESCRIPTION
**Pull request by Godfrey-Alimony**
Add Markup File of the Mobile work section portfolio.
Add Styling File of the  Mobile work section portfolio.
Add README File of the  Mobile work section portfolio.
Add MIT File of the  Mobile work section portfolio.
Add .gitignore File of the  Mobile work section portfolio.
Add Image Folder of the  Mobile work section portfolio.
Add .hintrc File of the  Mobile work section portfolio.
Add .stylelintrc File of the  Mobile work section portfolio.

### CODE REVIEW

File: portfolio.html [hunk 1]
```diff
@@ -72,6 +72,264 @@ <h1 class="Portfolio-hero-title">I’m Alimony Godfrey Glad<br>to see you!
...
+                <div class="Aboutme-flex16 layout">
+                    <div class="Aboutme-hero-title-box layout">
+                        <h3 class="Aboutme-hero-title">
+                            About<br>Myself</h3></div>
+            <h5 class="Aboutme-highlights2 layout">
+              Hello I’m a software developer! I can help you build a product ,
+              feature or website Look through some of my work and experience! If
+              you like what you see and have a project you need coded, don’t
+              hestiate to contact me.
+            </h5>
```
Comment (m05e5) [inline portfolio.html:247]: - [ ] Hi. Great job. it's recommended to use semantic HTML for this text ill recommend using a `<p>` tag instead of h5

**Review by m05e5 — CHANGES_REQUESTED** [review]
Hi @Godfrey-Alimony , 

Good job so far. I really like what you did!
There are some issues that you still need to work on to go to the next project but you are almost there!
mario_ladders


## Highlights


## Required Changes ♻️
* You really did a great job but there are some features you need to improve


- [ ] _**Please check the comments below**_

_I'm sure you can do it just keep it up_

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_@m05e5_

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: portfolio.html (hunk 1 above)
Comment (Godfrey-Alimony) [reply portfolio.html:247]: I have corected it Sir.Thank

**Comment by Godfrey-Alimony** [comment]
Dear Sir i have corected what you told me to corect .thank

**Review by zainsadaqat — APPROVED** [review]
# APPROVED!!!! 🎉 🎉 🎉 🎉 🎉 🎉
## Hey @Godfrey-Alimony  😊 😃 

#### Great Job 😄 👏 You've done it. Your project is complete!  There is nothing else to say other than... it's time to merge it 👍🏻 ✅ 

# Congratulations!!! 🎉 🥳 👏 🙌 🎊 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Godfrey-Alimony** [comment]
Thank you sir

**Pull request merged**